### DISCUSSION TITLE
Next.js on a Raspberry 4

### DISCUSSION THREAD

**Discussion by j-o-o** [vercel/next.js#20785] [Help]
Hello Internet,
I was wondering if it would be possible to deploy a Next.js App on a Raspberry. Just as a quarantine project.
w3schools says that a Raspberry can run Node.js https://www.w3schools.com/nodejs/nodejs_raspberrypi.asp
and I found out that you can actually deploy Next.js on a custom node.js server https://nextjs.org/docs/deployment 
It needs a custom server https://nextjs.org/docs/advanced-features/custom-server

I never tried anything like hosting on a Raspberry or have a custom server. Could that work? Or is it wasted time?
My next question would be if I could connect a MongoDB on the same machine, but thats another story I guess.

Looking forward!

**Comment by y0100100012** [comment]
I have tried it on my raspberry 4B.
It worked and nothing defferent from my x86 win10 pc

**Comment by michrome** [comment]
Yes! You can run Node, Next.js and MongoDB on a Raspberry Pi. 😀
